packages/react-core/src/demos/DescriptionList/examples/DescriptionListBasic.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  Button,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  PageSection,
+  PageSectionVariants,
+  Card,
+  CardBody
+} from '@patternfly/react-core';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+
+export const DescriptionListBasic: React.FunctionComponent = () => (
+  <DashboardWrapper>
+    <PageSection variant={PageSectionVariants.light}>
+      <Card>
+        <CardBody> Details</CardBody>
+      </Card>
+      <Card>
+        <CardBody>
```

Comment:
There should only be a single card on the page, with the "Details" text being placed inside a CardHeader using a Title component (as a level h2 heading), and the DescriptionList content placed in the CardBody.